### PR TITLE
Release v0.9.6 (windows build support)

### PR DESCRIPTION
**Pull request by vtermanis**
Have tested under Windows using VS 2015 & VS 2010 tools as detailed [here](https://wiki.python.org/moin/WindowsCompilers) so hopefully no longer need to exclude Windows builds.

### CODE REVIEW

**Comment by vtermanis** [comment]
Hi @ocefpaf - I only just realised that of course due to lack of an `appveyor.yml` presumably no Windows test build(s) will run. What should I do to enable Windows builds? (I [have made](https://github.com/Iotic-Labs/py-lz4framed/commit/bcc87ee10b3508b33b5c317755a044a3e74d7ee5) windows-specific fixes which should hopefully let the build succeed.)

**Comment by vtermanis** [comment]
@conda-forge/core - as a feedstock contributor, what is the right way to introduce a Windows build into this existing feedstock? Thank you in advance.

**Comment by scopatz** [comment]
You need to do a `conda smithy rerender` after you update the recipe.

**Comment by ocefpaf** [comment]
> You need to do a `conda smithy rerender` after you update the recipe.

That is recommended but this feedstock is working and there is no real functional change in the latest re-renderings. So to save CI time it is OK to merge this as is and rerender at a later time.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,7 +15,6 @@ source:
 
 build:
   number: 0
-  skip: True  # [win]
```
Comment (ocefpaf) [inline recipe/meta.yaml:18]: Ah. I missed that! @scopatz is correct! You do need to re-render as there is a functional change to the feedstock that needs to be applied here. (Sorry for the confusion.)

**Comment by vtermanis** [comment]
Thank you @scopatz  & @ocefpaf . Just so I understand (I haven't used `conda smithy` before - this is my first feedstock):
1. I merge this into the master branch
2. Just run `conda smithy rerender`

Do I not have to do anything about appveyor token(s) setup?

**Comment by ocefpaf** [comment]
First take a look at: https://github.com/conda-forge/staged-recipes/wiki/conda-smithy-rerender

There is no need to merge this now. Re-render and commit the changes to this PR. Everything is setup automatically by conda-smithy, don't worry about tokens or anything.

**Comment by vtermanis** [comment]
@ocefpaf - thank you! I think I've done the right steps and the Windows build seems to work (and tests pass). The Travis builds say `Waiting to be queued` - does that just mean the build system is busy and it'll happen later?

> There is no need to merge this now

Are you saying that once the Travis build passes you'll merge this PR into master and I should just leave it?

Thank you for all your (and your colleagues') help.

**Comment by ocefpaf** [comment]
> The Travis builds say Waiting to be queued - does that just mean the build system is busy and it'll happen later?

Yes. Even though it is always recommended to wait for the CIs you can merge this now since the change you performed was only to add the Windows builds.

> Are you saying that once the Travis build passes you'll merge this PR into master and I should just leave it?

No. I meant that you could add the re-rendering to this PR and merge this :smile: 

BTW feel free to go ahead and merge it now. Self-merges are fair game in the feedstock, but when in doubt feel free to bug us.

**Pull request merged**